backends/apple/mps/partition/mps_partitioner.py

```diff
@@ -30,8 +31,15 @@
 class MPSOperatorSupport(OperatorSupportBase):
     def __init__(self, edge_program: torch.export.ExportedProgram, compiler_specs):
         self.node_visitors = get_node_visitors(edge_program)
+        self.edge_program = edge_program
 
     def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
+        # Parameters are supported if any of their users are supported
+        if is_parameter(self.edge_program, node):
```

Comment:
Actually, after reading the pr again, why is the following lines mean it's parameter?

Reply:
here by it's parameter we're checking only if it's a constant (lifter buffer)

Reply:
>  it's a constant (lifter buffer)

right, I'm trying to find the logic to determine if it's a constant. This line doesn't look like so.

Reply:
We're checking if it's a constant if `is_param` or `is_buffer` are true (or `is_get_attr` is set to true - which I think it's not the case  anymore after moving to lifted graphs)

```
def is_parameter(exp_prog: torch.export.ExportedProgram, node: torch.fx.Node) -> bool:
    """
    Check if a node is a lifted parameter (static data like weights and bias are
    are supplied as inputs to the graph.

    Args:
        edge_program (torch.export.ExportedProgram): _description_
        node (torch.fx.Node): _description_

    Returns:
        bool: _description_
    """
    return is_get_attr(node) or is_param(exp_prog, node) or is_buffer(exp_prog, node)
```